napari/utils/info.py

```diff
@@ -211,15 +211,32 @@ def sys_info(as_html: bool = False) -> str:
         except PackageNotFoundError:
             text += f'  - {name} not installed<br>'
 
-    text += '<br><b>Settings path:</b><br>'
     try:
         from napari.settings import get_settings
 
-        text += f'  - {get_settings().config_path}'
+        _async_setting = str(get_settings().experimental.async_)
+        _autoswap_buffers = str(get_settings().experimental.autoswap_buffers)
+        _triangulation_backend = str(
+            get_settings().experimental.triangulation_backend
+        )
+        _config_path = get_settings().config_path
     except ValueError:
         from napari.utils._appdirs import user_config_dir
 
-        text += f'  - {os.getenv("NAPARI_CONFIG", user_config_dir())}'
+        _async_setting = str(os.getenv('NAPARI_ASYNC', 'False'))
+        _autoswap_buffers = str(os.getenv('NAPARI_AUTOSWAP', 'False'))
+        _triangulation_backend = str(
+            os.getenv('NAPARI_TRIANGULATION_BACKEND', 'Fastest available')
+        )
+        _config_path = os.getenv('NAPARI_CONFIG', user_config_dir())
```

Comment:
Why we need this ValueError path?

Reply:
I dunno, you added it 🤣 
I'm not sure what case triggers it, maybe CI?

Reply:
Yup, confirm it was added in #5333. That discussion was fun to read a few years later 😊 but offered no clues as to why this try/except is here. 😕 I agree with @psobolewskiPhD that it's probably best not to mess with it in this PR.